### DISCUSSION TITLE
Report type GET_FBA_SNS_FORECAST_DATA return status CANCELLED

### DISCUSSION THREAD

**Discussion by krystian-rzepa** [amzn/selling-partner-api-models#3714] [General]
Hi
After run createReport with reportType: "GET_FBA_SNS_FORECAST_DATA", status is changing from IN_QUEUE -> IN_PROGRESS -> CANCELLED. Other report types like GET_FBA_INVENTORY_PLANNING_DATA works fine with my config. I have products and orders on FBA. Do I have to provide any additional parameters or meet additional requirements to get this report?

**Comment by moborji** [comment]
same kind of issue for this type when I am going to getReportDocument($documentId, $reportType).
result: {
    "error": "Report type 'GET_FBA_SNS_FORECAST_DATA' is not supported"
}

**Reply by mafge** [reply]
Hi @moborji,
please feel free to reach out to [developer support](https://developer.amazonservices.com/support) to get help!
Best,
Marc
Selling Partner Developer Services
